app/src/main/res/values/themes.xml

```diff
@@ -61,4 +61,9 @@
   <style name="OppiaActionBarTheme" parent="ThemeOverlay.MaterialComponents.Dark.ActionBar" >
     <item name="android:background">@color/component_color_shared_activity_action_bar_color</item>
   </style>
+
+  <style name="OppiaAdminpinTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar" >
```

Comment:
"OppiaAdminpinTheme" is not correctly formatted. "P"in should be in uppercase

Reply:
<style name="OppiaAdminpinTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar" > 
 
this seems redundant code so I deleted